scripts/build_podspec.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the Swift Metrics API open source project
+##
+## Copyright (c) 2019 Apple Inc. and the Swift Metrics API project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of Swift Metrics API project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftNIO open source project
+##
+## Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+set -eu
+
+function usage() {
+  echo "$0 [-u] version"
+  echo
+  echo "OPTIONS:"
+  echo "  -u: Additionally upload the podspec"
+}
+
+upload=false
+while getopts ":u" opt; do
+  case $opt in
+    u)
+      upload=true
+      ;;
+    \?)
+      usage
+      exit 1
+      ;;
+  esac
+done
+shift "$((OPTIND-1))"
+
+if [[ $# -eq 0 ]]; then
+  echo "Must provide target version"
+  exit 1
+fi
+
+version=$1
+name="SwiftMetrics"
+
+here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+tmpdir=$(mktemp -d /tmp/.build_podspecsXXXXXX)
+echo "Building podspec in $tmpdir"
+
+  cat >> "${tmpdir}/${name}.podspec" <<- EOF
+Pod::Spec.new do |s|
+  s.name = '$name'
+  s.version = '$version'
+  s.license = { :type => 'Apache 2.0', :file => 'LICENSE.txt' }
+  s.summary = 'A Metrics API for Swift.'
+  s.homepage = 'https://github.com/apple/swift-metrics'
+  s.author = 'Apple Inc.'
+  s.source = { :git => 'https://github.com/apple/swift-metrics.git', :tag => s.version.to_s }
+  s.documentation_url = 'https://apple.github.io/swift-metrics'
+  s.module_name = '$name'
+
+  s.swift_version = '4.2'
```

Comment:
Should this section match SwiftNIO's? https://github.com/apple/swift-nio/blob/master/scripts/build_podspecs.sh#L88

Reply:
nio v. 2.0 needs swift 5, this one is okay with > 4.2